src/Engine/include/OpenLoco/Engine/World.hpp

```diff
@@ -63,6 +63,26 @@ namespace OpenLoco::World
         return validTileCoord(coords.x) && validTileCoord(coords.y);
     }
 
+    constexpr bool coordWithinMapBoundaries(const coord_t coord)
+    {
+        return (coord >= kTileSize) && (coord < (kMapWidth - kTileSize));
```

Comment:
is this any different to `drawableCoord`

Reply:
Hadn't noticed that one! Seems one unit different, but I'd be fine with the substitute. For reference:

```cpp
    constexpr bool drawableCoord(const coord_t coord)
    {
        return (coord >= (World::kTileSize - 1)) && (coord < (World::kMapWidth - World::kTileSize));
    }
```